src/tooltip.js

```diff
@@ -217,6 +218,18 @@ angular.module('ts.utils')
           }
         }
         else{
+          angular.element($window).on('resize', function(){
+            positionTooltip();
```

Comment:
I still think this will leak. It's a global listener on window added for every tooltip and doesn't seem to be removed.